Lib/test/test_typing.py

```diff
@@ -554,6 +554,12 @@ def test_many_weakrefs(self):
                     vals[x] = cls(str(x))
                 del vals
 
+    def test_constructor(self):
+        T = TypeVar(name="T")
```

Comment:
I find this form offensive, but since we allowed it in 3.11 I guess we have to keep supporting it, and we can't just make it an error (not crashing of course). (It's especially weird since a type var with constraints *requires* the name to be positional.)